ui/app/src/state/epics/pathNavigator.ts

```diff
@@ -392,16 +440,21 @@ export default [
         } = action;
         const parentOfTargetPath = getParentPath(targetPath);
         const parentOfSourcePath = getParentPath(sourcePath);
+        const idsToRefresh = [];
         Object.values(state.pathNavigator).forEach((navigator) => {
           if (navigator.isRootPathMissing && targetPath === navigator.rootPath) {
-            actions.push(pathNavigatorRefresh({ id: navigator.id }));
+            idsToRefresh.push({ id: navigator.id });
           } else if (!navigator.isRootPathMissing && navigator.currentPath.startsWith(sourcePath)) {
             actions.push(pathNavigatorSetCurrentPath({ id: navigator.id, path: navigator.rootPath }));
-          } else if (navigator.currentPath === parentOfTargetPath || navigator.currentPath === parentOfSourcePath) {
-            actions.push(pathNavigatorBackgroundRefresh({ id: navigator.id }));
+          } else if (
```

Comment:
While testing I was facing an issue when moving content that is not happening in 4.1 or 4.2, I found that there was a ticket/fix done for 4.1 (now 4.1 and 4.2) but not in 4.0. See ticket in [here](https://github.com/craftercms/craftercms/issues/6068)

Reply:
Noted the ticket reference but don't understand why the removal of the condition?

Reply:
I didn't remove the condition, see on lines 450 and 451, I wrapped `navigator.currentPath` with `withoutIndex`